### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by drkennetz**
Extensive changes to README to give a walk-through of installing a slack bot. Minor changes to main.

### CODE REVIEW

**Comment by drkennetz** [comment]
@hemantsoni42 would you like to review? If not, I can squash and merge. I don't mind either way. You don't need to review the changes to any of the go code, but you may want to look over the README.

**Comment by hemantsoni42** [comment]
@drkennetz  Sure I will do that.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    <a href="https://github.com/drkennetz/go-slack-chat-gpt3/graphs/contributors" alt="Contributors">
+        <img src="https://img.shields.io/github/contributors/drkennetz/go-slack-chat-gpt3.svg" /></a>
+    <a href="https://github.com/drkennetz/go-slack-chat-gpt3/pulse" alt="Activity">
+        <img src="https://img.shields.io/github/commit-activity/m/drkennetz/go-slack-chat-gpt3" /></a>
+    <a href="https://github.com/drkennetz/go-slack-chat-gpt3">
+        <img alt="GitHub Workflow Status" src="https://img.shields.io/github/actions/workflow/status/drkennetz/go-slack-chat-gpt3/ci.yml">
+    </a>
+<!--    <a href="https://github.com/drkennetz/go-slack-chat-gpt3">
+        <img src="https://img.shields.io/github/workflow/status/drkennetz/go-slack-chat-gpt3/Service%20Testing/main" alt="Service Testing Status">
+    </a> -->
```
Comment (hemantsoni42) [inline README.md:12]: @drkennetz can you please remove these commented line. They are created by me by mistake while I am adding badges to README.md file.

**Comment by drkennetz** [comment]
> @drkennetz Sure I will do that.

Thanks! I'll wait til you're done with the initial review before I commit changes. I appreciate the help!

File: README.md (hunk 1 above)
Comment (drkennetz) [reply README.md:12]: @hemantsoni42 just a note, if you have finished the review - click "request changes" so I know that I can begin work.

**Comment by drkennetz** [comment]
closes #10

**Comment by hemantsoni42** [comment]
@drkennetz my reviewing is completed. I only find some extra comment in README.md file which I was already mention above.

**Review by hemantsoni42 — CHANGES_REQUESTED** [review]
I completed my review you can begin your work.

**Comment by drkennetz** [comment]
@hemantsoni42 oddly enough, if I remove the commented lines the format changes and some stuff is lost. I'll merge it, and if you'd like to modify in a separate PR you can. Thanks for looking into that.

**Pull request merged**